packages/web/test/mocks/myalgowallet-mock.ts

```diff
@@ -54,10 +48,17 @@ export default class MyAlgoConnectMock {
 
     signTransaction(transaction: AlgorandTxn | EncodedTransaction | (AlgorandTxn | EncodedTransaction)[], signOptions?: SignTransactionOptions): Promise<SignedTx | SignedTx[]> {
         return new Promise((resolve, reject) => {
-            return resolve([{
-                blob: new Uint8Array(),
-                txID: ""
-            }]);
+            if (Array.isArray(transaction)) {
+                const signedTransaction = []
+                for (const txn of transaction) {
+                    const decodedtransaction = decodeUnsignedTransaction(txn as unknown as Uint8Array);
```

Comment:
can we directly set it to Uint8Array?

Reply:
do you mean `txn as Uint8Array`?
no we can't since the reader reads the txn with another type (their can be multiple values of it as per the parameter type)

Reply:
let me see if it is possible, i remember it showed me some error

Reply:
oh yes, it worked, thanks